src/emcpy/plots/plots.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+
+
+__all__ = ['Scatter', 'Histogram', 'LinePlot']
+
+
+class Scatter:
+
+    def __init__(self, x, y):
+        """
+        Constructor for Scatter.
+
+        Args:
+            x : (array type)
+            y : (array type)
+        """
+
+        self.plottype = 'scatter'
+
+        self.x = x
+        self.y = y
+
+        self.markersize = 5
+        self.color = 'darkgray'
+        self.marker = 'o'
+        self.vmin = None
+        self.vmax = None
+        self.alpha = None
+        self.linewidths = 1.5
+        self.edgecolors = None
+        self.label = f'n={np.count_nonzero(~np.isnan(x))}'
+
+    def add_linear_regression(self):
```

Comment:
Oh okay, I see the issue from my above question. Can we make these default to `False` so that the if statements are simpler?

Reply:
So I thought the same thing, but say if density scatter is False, I would also need to change sort and colorbar to false. That is why I added the methods that add them to the object. Do you think I should have linear regression and density set to False in the init so they are there and then keep the methods so all those options don't need to change?

Reply:
If I understand correctly, yes that is what I'm proposing. The init should have the defaults of `False` and keep your `add_linear_regression` to change them to `True`. Those specific attributes are fine I think to only add later especially if they only depend on something being `True`.

Reply:
Got it, will change!